Lib/test/test_tcl.py

```diff
@@ -144,26 +144,16 @@ def testUnsetVarException(self):
 
     def get_integers(self):
         integers = (0, 1, -1, 2**31-1, -2**31, 2**31, -2**31-1, 2**63-1, -2**63)
-        # bignum was added in Tcl 8.5, but its support is able only since 8.5.8.
-        # Actually it is determined at compile time, so using get_tk_patchlevel()
-        # is not reliable.
-        # TODO: expose full static version.
-        if tcl_version >= (8, 5):
-            v = get_tk_patchlevel()
-            if v >= (8, 6, 0, 'final') or (8, 5, 8) <= v < (8, 6):
-                integers += (2**63, -2**63-1, 2**1000, -2**1000)
+        integers += (2**63, -2**63-1, 2**1000, -2**1000)
```

Comment:
If a condition is unlocked, maybe there is a sense to group numbers into (N-1, -N, N, -N-1).

```python
    def get_integers(self):
        return (0, 1, -1,
                2**31-1, -2**31, 2**31, -2**31-1,
                2**63-1, -2**63 ,2**63, -2**63-1,
                2**1000, -2**1000)
```

Reply:
I thought about this, but kept simpler diff because there is already many changes in this PR.

But if you also suggest this I'll rewrite this code.